examples/shiny-deploy.yaml

```diff
@@ -0,0 +1,33 @@
+# Workflow derived from https://github.com/r-lib/actions/tree/master/examples
+# Need help debugging build failures? Start at https://github.com/r-lib/actions#where-to-find-help
+on:
+  push:
+    branches: [main, master]
+
+name: shiny-deploy
+
+jobs:
+  pkgdown:
```

Comment:
For consistency I think you need to change this `pkgdown` jobs name to `shiny-deploy` (am guessing a copy-paste from using that yaml).

Reply:
Thanks. Good catch. Fixed.